action.yml

```diff
@@ -0,0 +1,55 @@
+name: "Run Dapr Conformance Tests"
+description: "Run Dapr Conformance Tests against a pluggable component"
+branding:
+  icon: "arrow-right-circle"
+  color: "gray-dark"
+inputs:
+  socket:
+    description: "The socket path."
+    required: true
+  metadata:
+    description: "The component metadata."
+    required: false
+  operations:
+    description: "The tested operations."
+    required: false
+  test_config:
+    description: "The additional test config."
+    required: false
+  type:
+    type: choice
+    description: "The component type."
+    required: true
+    options:
+      - state
+      - pubsub
+      - inputbinding
+      - outputbinding
```

Comment:
Should those be `binding.input` and `binding.output`?

Reply:
that makes sense, will change, thanks!